src/renderer/src/components/song/context-menu/SongContextMenuItem.tsx

```diff
@@ -18,12 +19,13 @@ const SongContextMenuItem: Component<SongContextMenuItemProps> = (props) => {
   });
 
   return (
-    <button
+    <Button
```

Comment:
This should not be a `Button` component, replace it with a regular "button".

The element is also way too big now.

<img width="747" alt="image" src="https://github.com/user-attachments/assets/161e2d38-1aa5-4520-a4fd-81ec360e06c8">

Meke it look like the other lists:

<img width="273" alt="image" src="https://github.com/user-attachments/assets/21695e9c-4a0c-4e0a-a947-1f9d6e0cee64">

Reply:
Hopefully we will be able to use the `<List />` here once that component is more flashed out, but just replacing the `<Button/>` and fixing the styles should be goo enough  for now.